mitodl/mit-open
# resource drawer UI fixes

### What are the relevant tickets?

Closes https://github.com/mitodl/hq/issues/4782



### Description (What does it do?)
This Pr fixes mobile display and updates the ocw logo in the learning resource drawer.

### Screenshots (if appropriate):
<img width="493" alt="Screenshot 2024-07-09 at 2 16 19 PM" src="https://github.com/mitodl/mit-open/assets/196425/5dbd05bf-6f7c-486e-b21c-8955a6f6fee5">
<img width="236" alt="Screenshot 2024-07-09 at 2 19 20 PM" src="https://github.com/mitodl/mit-open/assets/196425/dea8b23a-b445-4121-b705-d5ce42836e2c">

### How can this be tested?
1. Checkout this branch.
2. go to any page that lists learning resources and click on one to bring up the details panel.
3. view the panel in mobile and try different content types to see how it looks
